stubs/opencv-python/cv2/cv2.pyi

```diff
@@ -0,0 +1,4551 @@
+    def create(self, *args, **kwargs): ...  # incomplete
+    def getDefaultName(self, *args, **kwargs): ...  # incomplete
+    def getDescriptorChannels(self, *args, **kwargs): ...  # incomplete
+    def getDescriptorSize(self, *args, **kwargs): ...  # incomplete
+    def getDescriptorType(self, *args, **kwargs): ...  # incomplete
+    def getDiffusivity(self, *args, **kwargs): ...  # incomplete
+    def getNOctaveLayers(self, *args, **kwargs): ...  # incomplete
+    def getNOctaves(self, *args, **kwargs): ...  # incomplete
+    def getThreshold(self, *args, **kwargs): ...  # incomplete
+    def setDescriptorChannels(self, dch) -> None: ...
+    def setDescriptorSize(self, dsize) -> None: ...
+    def setDescriptorType(self, dtype) -> None: ...
+    def setDiffusivity(self, diff) -> None: ...
+    def setNOctaveLayers(self, octaveLayers) -> None: ...
+    def setNOctaves(self, octaves) -> None: ...
+    def setThreshold(self, threshold) -> None: ...
+
+class AffineFeature(Feature2D):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def create(self, *args, **kwargs): ...  # incomplete
+    def getDefaultName(self, *args, **kwargs): ...  # incomplete
+    def getViewParams(self, tilts, rolls) -> None: ...
+    def setViewParams(self, tilts, rolls) -> None: ...
+
+class AgastFeatureDetector(Feature2D):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def create(self, *args, **kwargs): ...  # incomplete
+    def getDefaultName(self, *args, **kwargs): ...  # incomplete
+    def getNonmaxSuppression(self, *args, **kwargs): ...  # incomplete
+    def getThreshold(self, *args, **kwargs): ...  # incomplete
+    def getType(self, *args, **kwargs): ...  # incomplete
+    def setNonmaxSuppression(self, f) -> None: ...
+    def setThreshold(self, threshold) -> None: ...
+    def setType(self, type) -> None: ...
+
+class Algorithm:
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def clear(self) -> None: ...
+    def empty(self, *args, **kwargs): ...  # incomplete
+    def getDefaultName(self, *args, **kwargs): ...  # incomplete
+    def read(self, fn) -> None: ...
+    def save(self, filename) -> None: ...
+    def write(self, *args, **kwargs): ...  # incomplete
+
+class AlignExposures(Algorithm):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def process(self, src, dst, times, response) -> None: ...
+
+class AlignMTB(AlignExposures):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def calculateShift(self, *args, **kwargs): ...  # incomplete
+    def computeBitmaps(self, *args, **kwargs): ...  # incomplete
+    def getCut(self, *args, **kwargs): ...  # incomplete
+    def getExcludeRange(self, *args, **kwargs): ...  # incomplete
+    def getMaxBits(self, *args, **kwargs): ...  # incomplete
+    @overload
+    def process(self, src, dst, times, response) -> None: ...
+    @overload
+    def process(src, dst) -> None: ...
```

Comment:
```suggestion
    def process(self, src, dst) -> None: ...
```

Verified the requirement argument count in the REPL

Reply:
> The tool you're using to generate the stubs may have a bug where it omits the self argument to the second overload of a method.

Indeed! I hadn't noticed that. I used mypy.stubgen. But I think it' more an issue in the docstring (which to my understanding is used to generate some signatures):
```py
>>> from cv2.cv2 import * 
>>> help(AlignMTB.process) 
Help on method_descriptor:

process(...)
    process(src, dst, times, response) -> None
    .



    process(src, dst) -> None
    .   @brief Short version of process, that doesn't take extra arguments.
    .
    .       @param src vector of input images
    .       @param dst vector of aligned images
```